intl/msg_hash_us.h

```diff
@@ -1443,11 +1443,11 @@ MSG_HASH(
    )
 MSG_HASH(
    MENU_ENUM_SUBLABEL_CRT_SWITCH_HIRES_MENU,
-   "Use high resolution menu"
+   "Use a high resolution menu betweeeon content. Allows use of other menu drivers"
    )
 MSG_HASH(
    MENU_ENUM_LABEL_VALUE_CRT_SWITCH_RESOLUTION_USE_CUSTOM_REFRESH_RATE,
-   "Use a custom refresh rate specified in the configuration file if needed."
+   " Custom Refresh Rate"
```

Comment:
Please mind the whitespace! No label in RA starts with whitespace, if I recall correctly.